### DISCUSSION TITLE
deploy in cpanel

### DISCUSSION THREAD

**Discussion by ikal00** [vercel/next.js#67815] [Help]
### Summary

![Screenshot 2024-07-16 222124](https://github.com/user-attachments/assets/1b1c6a8a-86c4-4b79-96d9-5be3be0fef80)
![Screenshot 2024-07-16 223144](https://github.com/user-attachments/assets/3846594b-47c0-416c-be97-2787f23fafbe)
![Screenshot 2024-07-16 223120](https://github.com/user-attachments/assets/90e118b4-0e18-4681-82aa-017462be1c0e)
![Screenshot 2024-07-16 223046](https://github.com/user-attachments/assets/1daec443-55af-4860-834d-2a440ec11416)

Is there a solution to the problem?

**Comment by ByAnthony** [comment]
Hi @ikal00 👋 I was in the same situation as you a few months ago. Did not find much documentation about Next.js with CPanel. Just wrote a [wee guide](https://github.com/ByAnthony/newzealandtunnellers/blob/c4059e7d6203a6803d3d610db0aa46b0c6df2b32/docs/deployment.md) which hopefully may help you 🙏
